src/pages/Product/Product.js

```diff
@@ -0,0 +1,335 @@
+                  <Circle alt="user face" src={comment.img_url} />
+                  <CommentFeed>
+                    <UserAndContent>
+                      <User>{comment.user}</User>
+                      <div>{comment.content}</div>
+                    </UserAndContent>
+                    <CommentInfo>
+                      <Time>{comment.date}</Time>
+                      <CommentHeart onClick={onClickCommentHeart}>
+                        <FaHeart className="far fa-heart"></FaHeart>
+                        <span>{comment.like}</span>
+                      </CommentHeart>
+                    </CommentInfo>
+                  </CommentFeed>
+                </Comment>
+              );
+            })}
+        </div>
+      </Article>
+      <Aside>
+        <div>
+          <LikeButton>
+            <FaHeart className="far fa-heart"></FaHeart>22
+          </LikeButton>
+          <Profile>
+            <UserInfo>
+              <Circle
+                alt="user face"
+                src="https://images.unsplash.com/photo-1506748686214-e9df14d4d9d0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80"
+              />
+              <Name>jiyon</Name>
+            </UserInfo>
+            <Follow>팔로우</Follow>
+          </Profile>
+        </div>
+        <Circle as="div" onClick={onClickURL}>
+          URL
+        </Circle>
+      </Aside>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+`;
+
+const Article = styled.div`
+  padding: 20px;
+  border-right: 1px solid ${props => props.theme.borderLine};
+`;
+
+const ProductHeader = styled.div`
+  ${flexSet('space-between', 'center')}
+  padding: 50px 0;
+  color: ${props => props.theme.fontGray};
+`;
+
+const ProductImgContainer = styled.div`
+  max-width: 800px;
+
+  &::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 1px;
+    margin: 30px 0;
+    background-color: ${props => props.theme.borderLine};
+  }
+`;
+
+const ProductImg = styled.img`
+  width: 100%;
+  object-fit: cover;
+`;
+
+const CommentCount = styled.div`
+  font-size: 18px;
+  font-weight: 700;
+  margin-bottom: 30px;
+`;
+
+const CommentCountNum = styled.span`
+  color: ${props => props.theme.mainBlue};
+`;
+
+const InputContainer = styled.div`
+  position: relative;
+  display: flex;
+`;
+
+const FaSmile = styled.div`
+  font-size: 1.4rem;
+  color: grey;
+  margin-right: 10px;
+`;
+
+const CommentInput = styled.input`
+  position: absolute;
+  top: -10px;
+  padding: 10px;
+  width: 95%;
+  border: 1px solid ${props => props.theme.inputGray};
+  border-radius: 5px;
+`;
+
+const Register = styled.button`
+  position: absolute;
+  right: 12px;
+  font-weight: 700;
+  color: ${props => props.theme.mainBlue};
+  ${props =>
+    props.alert
+      ? css`
+          opacity: 0.6;
+        `
+      : css`
+          opacity: 1;
+        `}
```

Comment:
props로 조건부 스타일링 하는거 너무 잘 적용해주셨네요!!
좀 더 깔끔하게 쓰자면 아래와 같이 쓸 수 있습니다.
```suggestion
  ${({theme,alert}) => css`
    color:${theme.mainBlue};
    opacity:${alert ? 0.6 : 1};
    `
```